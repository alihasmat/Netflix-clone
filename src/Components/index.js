export { default as Accordian } from "./Accordian/Accordian";
export { default as Card } from "./Card/Card";
export { default as Feature } from "./Feature/Feature";
export { default as Footer } from "./Footer/Footer";
export { default as Form } from "./Form/Form";
export { default as Header } from "./Header/Header";
export { default as Jumbotron } from "./Jumbotron/Jumbotron";
export { default as Loading } from "./Loading/Loading";
export { default as Player } from "./Player/Player";
export { default as Profiles } from "./Profiles/Profiles";
export { default as Search } from "./Search/Search";
