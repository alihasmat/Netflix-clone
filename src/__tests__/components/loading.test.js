import { render } from "@testing-library/react";
import { Loading } from "../../Components";

describe("<Loading />", () => {
  it(`renders <Loading />`, () => {
    const { container, getByTestId } = render(
      <Loading src="/images/Hasmat.png" data-testid="loading" />
    );
    expect(getByTestId("loading")).toBeTruthy();
    expect(getByTestId("loading-picture")).toBeTruthy();
    expect(container.firstChild).toMatchSnapshot();
  });
  it(`renders <loading /> releasebody`, () => {
    const { container } = render(<Loading.ReleaseBody data-testid="loading" />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
