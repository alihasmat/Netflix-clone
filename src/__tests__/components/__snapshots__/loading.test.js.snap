// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> renders <Loading /> 1`] = `
<div
  class="style__Spinner-sc-xp1n64-0 buHoEw"
  data-testid="loading"
>
  <img
    alt="spinner"
    class="style__LoadingPicture-sc-xp1n64-1 kkFdfr"
    data-testid="loading-picture"
    src="images/users//images/Hasmat.png.png"
  />
</div>
`;

exports[`<Loading /> renders <loading /> releasebody 1`] = `null`;
